code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -270,7 +270,7 @@
 	id = "gygax_armor"
 	build_type = MECHFAB
 	build_path = /obj/item/mecha_parts/part/gygax_armor
-	materials = list(MAT_METAL=25000,MAT_DIAMOND=10000)
+	materials = list(MAT_METAL=10000,MAT_DIAMOND=10000,MAT_TITANIUM=15000)
```

Comment:
I was the person who didn't want to make big balance changes now you're complaining goddamnit